mirrord/sip/src/lib.rs

```diff
@@ -503,7 +506,19 @@ mod main {
     /// SIP check for binaries.
     fn is_binary_sip(path: &Path, data: &[u8], opts: SipPatchOptions) -> Result<bool> {
         // Skip patching binary if it is in the list of binaries to skip.
-        if opts.skip.iter().any(|x| path.ends_with(x)) {
+        if opts
+            .skip
+            .as_ref()
+            .unwrap_or(
```

Comment:
I like that the user-specified list replaces and is not added to the default list (because otherwise the user wouldn't be able to choose to not skip executables from the default list), but it also means it's a potential footgun for users that as we anticipate need the defaults, but also need additional values, because it would be easy to just add the additional values, and then lose the defaults. So we should definitely document the default behaviour and the effect of specifying values in the config docs.
And then this adds another reason to set the default at the config definition and not at the usage - the docs and the defaults are more likely to go out of sync if they are defined in different places.

Reply:
Yeah, agreed, I will move it to the config crate and mention to append to the default value instead of replacing it unless to intentionally patch some of the default list of processes.